packages/checkout/sdk/src/utils/utils.ts

```diff
@@ -1,3 +1,9 @@
+import { ChainId } from '../types';
+
 export const isMatchingAddress = (addressA: string = '', addressB: string = '') => (
   addressA.toLowerCase() === addressB.toLowerCase()
 );
+
+export const isZkEvmChainId = (chainId: ChainId) => chainId === ChainId.IMTBL_ZKEVM_DEVNET
```

Comment:
What do you think about this living in the `/network/network.ts` file? I worry utils becomes too generic as it grows.

Reply:
network is focused on switching/adding networks etc and this isn't being used for those functions. it could live clsoer to where its used - `transaction/transaction.ts`